client/app/vps/dashboard/vps-dashboard.controller.js

```diff
@@ -66,6 +66,7 @@ class VpsDashboardCtrl {
         this.VpsService.getTabSummary(this.serviceName, true)
             .then(summary => {
                 this.summary = summary;
+                this.initBackupStorageActions();
```

Comment:
I don't think this needs to be done in .then ;).

It's a detail tho.

Reply:
also, if action availability is dependant of this.loaders.summary = false I'd expect to see it in action.isAvailable

Reply:
I wanted to load all actions at the same place in the controller. BackupStorageAction is not using the summary but SnapshotActions is. It's a case of a controller who do more than one thing, we should check to separate functionalities...

Reply:
I working on a sepatate branch on this issue, I will create a new service to provide all actions for the dashboard.